testData/preferences/mike.json5

```diff
@@ -36,7 +36,9 @@
                         "EnhancePrecision": true,
                         "SnapToDefaultButton": 1,
                         "HidePointer": 0,
-                        "MouseSonar": 1
+                        "MouseSonar": 1,
+                        "DoubleClickHeight": 32,
```

Comment:
Can you add these two to the list of settings above? Also, s/Simon/Mike

Reply:
Done